docroot/themes/humsci/humsci_basic/src/scss/components/_masthead.scss

```diff
@@ -1,3 +1,46 @@
-.su-masthead .su-lockup {
-  z-index: $hb-z-index-lockup;
+.su-masthead {
+  .su-lockup {
+    z-index: $hb-z-index-lockup;
+  }
+
+  .su-site-search {
+    display: block;
+    padding-top: hb-calculate-rems(12px);
+    padding-bottom: hb-calculate-rems(28px);
+    max-width: hb-calculate-rems(265px);
+    z-index: $hb-z-index-2;
+  }
+
+  .su-main-nav {
+    padding-bottom: hb-calculate-rems(14px);
+
+    .su-main-nav__toggle {
+      position: absolute;
+      right: 0;
+      bottom: 80%;
+      transform: translate(0, -80%); // Center the toggle on nav styles implementation. It is not centered while using Decanter's default nav styles because they overlay ontop of the lockup which makes the content inaccessible.
+
+      @include grid-media-min('sm') {
+        bottom: 60%;
+        transform: translate(0, -60%);
+      }
+    }
+
+    // The positioning of the nav set here will change as Decanter overlays the mobile nav on top of other content and our designs have the mobile nav inline with the content.
```

Comment:
can you break this comment into two lines so it is a more readable line length? thanks!